### PR TITLE
Optimizing existing models

### PR DESCRIPTION
**Pull request by daanraman**

### CODE REVIEW

File: app/analyzers/terms.py [hunk 1]
```diff
@@ -124,8 +91,47 @@ def evaluate_model(self):
...
+            else:
+                num_docs_processed += 1
+
+        logging.logger.info("going to brute force " + str(len(field_names_to_brute_force)) + " fields")
+        return field_names_to_brute_force
+
     def extract_additional_model_settings(self):
         self.model_settings["target"] = settings.config.get(self.config_section_name, "target").replace(' ', '').split(",")  # remove unnecessary whitespace, split fields
+
+        if "*" in self.model_settings["target"]:
```
Comment (michielmeersmans) [inline app/analyzers/terms.py]: You could write this as a single line:
```
self.model_settings["brute_force_target"] = "*" in self.model_settings["target"]
```

File: app/helpers/housekeeping.py [hunk 2]
```diff
@@ -24,18 +24,19 @@ def run(self):
...
+                    logging.logger.info("housekeeping - going to remove all whitelisted outliers")
                     total_docs_whitelisted = es.remove_all_whitelisted_outliers()
 
                     if total_docs_whitelisted > 0:
                         logging.logger.info("housekeeping - total whitelisted documents cleared from outliers: " + str(total_docs_whitelisted))
                     else:
                         logging.logger.info("housekeeping - whitelist did not remove any outliers")
 
-                except Exception as e:
+                except Exception:
```
Comment (michielmeersmans) [inline app/helpers/housekeeping.py]: Can you send this exception to sentry too?
```python
from sentry_sdk import capture_exception

try:
    a_potentially_failing_function()
except Exception as e:
    # Alternatively the argument can be omitted
    capture_exception(e)
```

Comment (daanraman) [reply app/helpers/housekeeping.py]: Tracked as issue, thanks

File: app/analyzers/terms.py (hunk 1 above)
Comment (daanraman) [reply app/analyzers/terms.py]: Changed, thanks!

**Pull request merged**